### PR TITLE
chore(copy): email invite confirmation dialog

### PR DESCRIPTION
**Pull request by Karim94**
#### Relevant info and issue/PR links 
https://app.clubhouse.io/peerio/story/7245/mobile-implement-email-confirmation-dialog-illustration
#### Testing instructions  
N/A

----
### Repository owner

- [ ] Was tested and can be merged.
- [ ] PR name follows conventional changelog format.

[PR guideline](https://github.com/PeerioTechnologies/peerio-icebear/blob/dev/docs/CONTRIBUTING.md)

### CODE REVIEW

**Comment by ebilg** [comment]
can be merged

File: src/copy/en.json [hunk 1]
```diff
@@ -90,6 +90,9 @@
   "title_contactsOnPeerio": "Contacts on Peerio",
   "title_deselectAll": "Deselect All",
   "title_selectAll": "Select All",
+  "title_confirmEmailInvitesHeading": "Confirm email invites",
+  "title_confirmEmailInvite": "Email invites will be sent to {numSelectedContacts} people.",
+  "title_confirmEmailInvite2": "Email invite will be sent to {numSelectedContacts} person.",
```
Comment (anri-asaturov) [inline src/copy/en.json]: `{numSelectedContacts} person.`?
can numSelectedContacts be something other then `1` when you use this line?

Comment (Karim94) [reply src/copy/en.json]: It can be 1 person or X people.

Comment (anri-asaturov) [reply src/copy/en.json]: Why do u need a variable if it's always `1`?

Comment (Karim94) [reply src/copy/en.json]: I see, will push a fix soon.

Comment (anri-asaturov) [reply src/copy/en.json]: Actually this is a bad way to do plurals, it only works for english. It's gonna be untranslatable. @ebilg

Comment (ebilg) [reply src/copy/en.json]: sorry my anglo brain missed this, I've asked Paul for better copy

**Comment by ebilg** [comment]
can this be merged for now since mobile branch is merged and I'll track the task to make sure copy is changed to be translate-able

**Pull request merged**